packages/frontend/components/scaffold-eth/Balance.tsx

```diff
@@ -0,0 +1,80 @@
+import { useState } from "react";
+import { useBalance } from "wagmi";
+
+type BalanceProps = {
+  address: string;
+  price: number;
+  minimized?: boolean;
+  wrapperClasses?: string;
+};
+/**----------------------
+ * Display balance of an address
+ * ---------------------*/
+
+export default function Balance({ address, price, minimized, wrapperClasses }: BalanceProps) {
+  const [isEthBalance, setIsEthBalance] = useState<boolean>(true);
+
+  const {
+    data: balanceData,
+    isError,
+    isLoading,
+  } = useBalance({
+    addressOrName: address,
+  });
+
+  let displayAddress = address?.slice(0, 5) + "..." + address?.slice(-4);
+
+  const onToggleBalance = () => {
+    setIsEthBalance(!isEthBalance);
+  };
+
+  //   on loading screen or no address supplied
+  if (!address || isLoading) {
+    return (
+      <div className="animate-pulse flex space-x-4">
+        <div className="rounded-md bg-slate-300 h-6 w-6"></div>
+        <div className="flex items-center space-y-6">
+          <div className="h-2 w-28 bg-slate-300 rounded"></div>
+        </div>
+      </div>
+    );
+  }
+
+  //on fetch balance error
```

Comment:
There is no space after //.

Also consider that we might not need these kind of comments. if (isError) is self explanatory :P Doesn't bother me if you want to leave it tho